app/models/site_config.rb

```diff
@@ -25,6 +25,9 @@ class SiteConfig < RailsSettings::Base
   # Authentication
   field :authentication_providers, type: :array, default: %w[twitter github]
 
+  # Broadcast
+  field :welcome_notifications_live_date, type: :date, default: "2030-01-01 00:00:00 UTC".to_date
```

Comment:
```suggestion
  field :welcome_notifications_live_at, type: :date, default: "2030-01-01 00:00:00 UTC".to_date
```

small nitpick, but i think "live_at" is a little more clear? what do you think? :)

Reply:
Yup that works for me